stats/measure_int64.go

```diff
@@ -44,3 +46,13 @@ func Int64(name, description, unit string) (*Int64Measure, error) {
 	}
 	return m, nil
 }
+
+// MustInt64 creates a new measure of type Int64Measure. It will log a
+// fatal error if a measure with the same name already exists.
```

Comment:
Should we panic rather than fatal like regexp.MustCompile, template.Must?

Reply:
@ramonza for sure let's panic then, thanks for this catch. My bad, I was the one who suggested https://github.com/census-instrumentation/opencensus-go/pull/479#discussion_r171699498 log.Fatalf but that was for the internal helper functions.